### PR TITLE
Try to get a high-res photo

### PR DESCRIPTION
**Pull request by MarsVard**
Try to get the high res image from the camera if possible. Otherwise fall back on low res images.

### CODE REVIEW

**Comment by MarsVard** [comment]
I signed it!

File: library/src/main/camera2/com/google/android/cameraview/Camera2.java [hunk 1]
```diff
@@ -422,9 +422,19 @@ private void collectCameraInfo() {
...
-        for (android.util.Size size : map.getOutputSizes(ImageFormat.JPEG)) {
-            mPictureSizes.add(new Size(size.getWidth(), size.getHeight()));
+        // try to get hi-res output sizes for Marshmellow and higher
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            for (android.util.Size size : map.getHighResolutionOutputSizes(ImageFormat.JPEG)) {
+                mPictureSizes.add(new Size(size.getWidth(), size.getHeight()));
+            }
+        }
+        // fallback camera sizes and lower than Marshmellow
+        if(mPictureSizes.ratios().size() == 0){
```
Comment (yaraki) [inline library/src/main/camera2/com/google/android/cameraview/Camera2.java]: Please format

Comment (MarsVard) [inline library/src/main/camera2/com/google/android/cameraview/Camera2.java]: Sorry about that, I just pushed my changes again, formatted this time.

Comment (yaraki) [inline library/src/main/camera2/com/google/android/cameraview/Camera2.java]: No, but could you format only the lines that you edited? Sorry for not being clear.

Comment (MarsVard) [inline library/src/main/camera2/com/google/android/cameraview/Camera2.java]: That's ok, done.

**Comment by yaraki** [comment]
Thank you so much for fixing this! I'm merging this change when I have my PC (I'm in an airport now).

**Comment by MarsVard** [comment]
You are very welcome! Thank you for the library! I hope to fix some more stuff soon.

**Pull request merged**